custom_components/healthchecksio/coordinator.py

```
@@ -0,0 +1,76 @@
+    from homeassistant.config_entries import ConfigEntry
+    from homeassistant.core import HomeAssistant
+
+_LOGGER: logging.Logger = logging.getLogger(__name__)
+
+
+class HealthchecksioDataUpdateCoordinator(DataUpdateCoordinator):
+    """Class to manage fetching Healthchecks.io data."""
+
+    config_entry: ConfigEntry
+
+    def __init__(
```

Pylint (3.3.7)

[refactor] Too many arguments (8/5)

(R0913)